### ISSUE TITLE
Adding assignees and tags

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#1451]
How do I add assignees and tags to issues in Git Hub?

**Comment by Jegelewicz** [comment]
Yes, it does, but it also means that most people are not able to make use of those functions. Can we let everyone have access to those functions?

**Comment by dustymc** [comment]
@Jegelewicz see http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

I have mixed feelings on this. I'd like to have more info from users (eg, priority) but I also think the AWG is ultimately responsible for that sort of thing. I can't tell "user says this is important" from "an actual review process says this is important." I don't think there's any way to reserve certain labels for one group of users or similar - eg, anyone who can assign any label can assign them all.

We should discuss who is an owner - I think they (or the person who somehow managed to get their password) can do fun things like delete repositories.

I'm adding this to the AWG agenda.

**Comment by Jegelewicz** [comment]
It's already on the agenda - thanks!

**Comment by dustymc** [comment]
https://github.com/ArctosDB/arctos/projects/6
